Source/Bricks/Image/ImageBrick.swift

```diff
@@ -184,7 +184,17 @@ open class ImageBrickCell: GenericBrickCell, Bricklike, AsynchronousResizableCel
             }
             
             imageLoaded = true
-        } else if let imageURL = dataSource.imageURLForImageBrickCell(self) {
+        }
+    }
+
+    override open func willDisplay() {
```

Comment:
I'd like to see if "apply(layoutAttributes" would do the trick instead. Then you don't rely on a delegate